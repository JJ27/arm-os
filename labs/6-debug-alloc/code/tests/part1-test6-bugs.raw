FOUND: </dev/ttyUSB0>
opened tty port </dev/ttyUSB0>.
pi-install: tty-usb=</dev/ttyUSB0> program=<./tests/part1-test6-bugs.bin> about to boot
TRACE:simple_boot: sending 13532 bytes, crc32=cacf1ee0
waiting for a start
putting code
bootloader: Done.
listening on ttyusb=</dev/ttyUSB0>
redzone checking: init
test4: use after free
TRACE:ck_heap_errors:going to check heap
TRACE:ck_heap_errors:SUCCESS: checked 6 blocks, detected no errors
TRACE:notmain:SUCCESS heap checked out
TRACE:notmain:corrupting [0x17ec20] at offset 173 [0x17eccd]
TRACE:ck_heap_errors:going to check heap
TRACE:mem_check:ERROR:Freed block 5 [0x17ec20] corrupted at offset 173
TRACE:hdr_print:	logical block id=5, [addr=0x17ec20] nbytes=171
TRACE:hdr_print:	Block allocated at: tests/part1-test6-bugs.c:notmain:14
TRACE:ck_heap_errors:checked 6 blocks, detected 1 errors
TRACE:notmain:SUCCESS found error
DONE!!!

Saw done

bootloader: pi exited.  cleaning up
