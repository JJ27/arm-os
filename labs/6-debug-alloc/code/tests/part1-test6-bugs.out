TRACE:ck_heap_errors:going to check heap
TRACE:ck_heap_errors:SUCCESS: checked 6 blocks, detected no errors
TRACE:notmain:SUCCESS heap checked out
TRACE:notmain:corrupting 
TRACE:ck_heap_errors:going to check heap
TRACE:mem_check:ERROR:Freed block 5  corrupted at offset 173
TRACE:hdr_print:	logical block id=5,  nbytes=171
TRACE:hdr_print:	Block allocated at: tests/part1-test6-bugs.c:notmain:14
TRACE:check_list:	Wrote block after free!
TRACE:ck_heap_errors:checked 6 blocks, detected 1 errors
TRACE:notmain:SUCCESS found error
